aea/helpers/ipfs/base.py

```diff
@@ -65,9 +64,17 @@ def _read(file_path: str) -> bytes:
     return file_b
 
 
+def chunks(data: Sized, size: int) -> Generator:
+    """Yield successivesize chunks from data."""
+    for i in range(0, len(data), size):
+        yield data[i : i + size]  # type: ignore
+
+
 class IPFSHashOnly:
     """A helper class which allows construction of an IPFS hash without interacting with an IPFS daemon."""
 
+    DEFAULT_CHUNK_SIZE = 262144
```

Comment:
Where is this coming from? Please add link to relevant documentation on IPFS side